_includes/head.html

```diff
@@ -18,11 +18,6 @@
   <meta name="og:description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
   <meta property="og:site_name" content="{{ site.description }}">
 
-  <!-- Social: Google+ / Schema.org  -->
```

Comment:
Do we want to remove this, or just edit the comment?

Reply:
probably just have `Schema.org` in the comment

Reply:
I think you still want to include all these `meta` tags under the Schema.org comment.

Reply:
Yeah, Ken is right. You didn't want to remove those meta tags as they power google search.

Reply:
Not really proposing this as a change right now, but at some point should we move away from inline schema and move toward `json-ld`? I think it's cleaner to have everything defined in the json object.

Reply:
Yeah, we might do that when we move to Hugo